react/week2/new-todo/src/Todolist.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import Todoitem from "./Todoitem";
+
+function Todolist({ todos }) {
+  const [itemList, setItemList] = useState(todos);
+
+  function addNewItem() {
+    const newItem = {
+      id: Math.floor(Math.random() * 100),
+      description: "add task",
+    };
+    setItemList(itemList.concat(newItem));
+  }
+    const deleteItem = (id) => {
+      const todos = [...itemList];
+      setItemList(todos.filter((todo) => todo.id !== id));
+      };
+  return (
+    <div>
+      <button onClick={addNewItem}>Add todo</button>
+      {itemList.length > 0 ? (
+        <ul>
+          {itemList.map((todo) => (
+            <Todoitem
+              key={todo.id}
+              id={todo.id + todo.description.slice(0, 2)}
+              description={todo.description}
```

Comment:
What is the purpose of adding description and then slice ?
you need only id here so use this   id={todo.id}

Reply:
Hi Swapna Thanks for reviewing my hw and giving a valuable Feedback..  I will looking in to it.